src/bin/julialauncher.rs

```diff
@@ -250,7 +248,7 @@ fn run_app() -> Result<i32> {
     }
 
     for (i, v) in args.iter().skip(1).enumerate() {
-        if i > 1 || !v.starts_with("+") {
+        if i > 1 || !v.starts_with('+') {
```

Comment:
Various places I've replaced single-char `str` with `char` for efficiency and simplicity.